eth/blockwatch/block_watcher.go

```diff
@@ -146,6 +146,25 @@ func (w *Watcher) InspectRetainedBlocks() ([]*MiniHeader, error) {
 	return w.stack.Inspect()
 }
 
+func (w *Watcher) syncToLatestBlock() error {
+	lastBlock, err := w.client.HeaderByNumber(nil)
```

Comment:
Should this be called lastHeader since HeaderByNumber returns a header?

Reply:
Adopted the naming from https://github.com/0xProject/0x-mesh/blob/80d4c33b9c753dd967dd653d6fbfb8f46dffc606/ethereum/blockwatch/block_watcher.go#L257

Reply:
Bump because it looks like the variable name here was not updated.